README.md

```diff
@@ -501,6 +504,158 @@ ASP.NET Integration
+
+   ```xml
+   <targets>
+     <target name="airbrake"
+             type="Airbrake"
+             projectId="113743"
+             projectKey="81bbff95d52f8856c770bb39e827f3f6"
+             environment="live"
+             ignoreEnvironments="dev"
+     />
+     <!-- other targets -->
+   </targets>
+   ```
+
+   ```xml
+   <rules>
+     <logger name="*" minlevel="Error" writeTo="airbrake" />
+     <!-- other rules -->
+   </rules>
+   ```
+
+   Note that both `projectId` and `projectKey` are required parameters. You can
+   set any configuration option supported by the Airbrake client in the declarative
+   way ([how to configure](#configuration)).
+
+   When you need to access `Notifier` programmatically (e.g. for setting up filters
+   in your code) you can get it from the `AirbrakeTarget` object:
+
+   ```csharp
+   var airbrakeTarget = (AirbrakeTarget)LogManager.Configuration.AllTargets
+       .FirstOrDefault(t => t.GetType() == typeof(AirbrakeTarget));
+   
+   airbrakeTarget?.Notifier.AddFilter(notice =>
+   {
+       // clear environment variables with "token"-related keys
+       new List<string>(notice.EnvironmentVars.Keys).ForEach(key =>
+       {
+           if (key.ToLowerInvariant().Contains("token"))
+               notice.EnvironmentVars[key] = "[removed]";
+       });
+   
+       return notice;
+   });
+   ```
+
+### Airbrake NLog target for ASP.NET
+
+Airbrake NLog target for ASP.NET differs from the regular one in the additional
+support for the HTTP context.
+
+1. Install the `Sharpbrake.NLog.Web` package from NuGet (you can use "Package
+   Manager Console" from Visual Studio):
+
+   ```
+   PM> Install-Package Sharpbrake.NLog.Web
+   ```
+
+2. Register the `Sharpbrake.NLog.Web` assembly in the `<extensions />` section
+   of the [nlog.config][nlog-config] file:
+
+   ```xml
+   <extensions>
+     <add assembly="Sharpbrake.NLog.Web"/>
+   </extensions>
+   ```
+
+3. Define the Airbrake target in the `<targets />` section and appropriate
+   routing rules in the `<rules />` section of `nlog.config`. All supported
+   settings are the same as for the regular Airbrake target.
+
+   ```xml
+   <targets>
+     <target name="airbrake"
+             type="Airbrake"
+             projectId="113743"
+             projectKey="81bbff95d52f8856c770bb39e827f3f6"
+             environment="live"
+             ignoreEnvironments="dev"
+     />
+     <!-- other targets -->
+   </targets>
+   ```
+
+   ```xml
+   <rules>
+     <logger name="*" minlevel="Error" writeTo="airbrake" />
+     <!-- other rules -->
+   </rules>
+   ```
+
+4. In the case if you are setting up the target for ASP.NET Core, extra step is
+   needed to get reporting of HTTP context properties. In the method `Configure`
+   of `Startup.cs` please call the next method:
+
+   ```csharp
+   app.ConfigureAirbrakeTarget();
+   ```
+
+   Here you have also access to the `Notifier` object that can be used for the
+   additional configuration from your code:
```

Comment:
Does the word "too" here imply that there is another place where the `Notifier` object can be accessed? Or it means "in addition to [setting NLog, some framework features, etc.] you can access Notifier as well"?